composer.json

```diff
@@ -9,18 +9,18 @@
             "homepage": "https://github.com/symfony-cmf/symfony-cmf/contributors"
         }
     ],
-    "minimum-stability":"dev",
+    "minimum-stability": "RC",
     "require": {
         "php": "^5.3.9|^7.0",
         "symfony/framework-bundle": "~2.3",
-        "doctrine/phpcr-bundle": "1.3.*",
-        "doctrine/phpcr-odm": "1.3.*",
+        "doctrine/phpcr-bundle": "~1.3",
+        "doctrine/phpcr-odm": "~1.3",
         "symfony-cmf/core-bundle": "1.3.*",
         "symfony-cmf/content-bundle": "1.3.*",
-        "symfony-cmf/menu-bundle": "2.*",
-        "symfony-cmf/block-bundle": "1.2.*",
+        "symfony-cmf/menu-bundle": "2.0.*",
+        "symfony-cmf/block-bundle": "1.3.*",
         "symfony-cmf/routing-bundle": "1.4.*",
-        "symfony-cmf/media-bundle": "1.3.*"
+        "symfony-cmf/media-bundle": "1.2.*"
```

Comment:
we are almost there for media 1.3. only open issue is https://github.com/symfony-cmf/MediaBundle/pull/148 - think we can wrap that up and then do a 1.3.0-RC1 of media bundle?